src/components/editMeasure/details/MeasureDefinitions/MeasureDefinitionRow.test.tsx

```diff
@@ -6,17 +6,24 @@ afterEach(cleanup);
 
 describe("Measure Definition Row Component", () => {
   it("Measure Definition Row renders at all", async () => {
-    const { getByText } = render(
+    const { getByText, getByTestId } = render(
       <MeasureDefinitionRow
         measureDefinition={{
           term: "term",
           definition: "I'm a measure definition",
         }}
+        setOpen={jest.fn()}
+        setSelectedDefinition={jest.fn()}
       />
     );
     const term = getByText("term");
     expect(term).toBeInTheDocument();
     const definition = getByText("I'm a measure definition");
     expect(definition).toBeInTheDocument();
+
+    const editButton = await getByTestId(
```

Comment:
`getByTestId` does not return a promise, so `await` is not needed.

Reply:
@jkotanchik-SB Thanks a lot Joe! Removed.

Reply:
This file was deleted.